Tella/Scenes/HomeView/Views/FileDetailView/FileDetailView.swift

```diff
@@ -30,18 +30,35 @@ struct FileDetailView: View {
         }
         .frame(maxWidth: .infinity, maxHeight: .infinity)
         .background(Styles.Colors.backgroundMain)
-        .fullScreenCover(isPresented: $isEditImagePresent) {
+        .fullScreenCover(isPresented: $isEditFilePresented) {
```

Comment:
What do you think of using UIKit presenting view (there is already an extension) ?